alf/algorithms/mi_estimator.py

```diff
@@ -74,7 +75,7 @@ class MIEstimator(Algorithm):
        samples from the two buffers.
     * 'shuffle': randomly shuffle batch y
     * 'shift': shift batch y by one sample, i.e.
-      tf.concat([y[-1:, ...], y[0:-1, ...]], axis=0)
+      torch.cat([y[-1:, ...], y[0:-1, ...]], dim=0)
```

Comment:
Can you update the entire docstring so that math and code strings will be properly displayed?

Reply:
updated